google_compute_engine/accounts/accounts_daemon.py

```diff
@@ -37,20 +37,30 @@ class AccountsDaemon(object):
   invalid_users = set()
   user_ssh_keys = {}
 
-  def __init__(self, groups=None, remove=False, debug=False):
+  def __init__(
+      self, groups=None, remove=False,
+      useradd_cmd='useradd -m -s /bin/bash -p * {user}',
```

Comment:
A clarification - can you pull them into variables defined at the top, set them to None here, and then when setting the class variables set them do something like:
```
self.utils = accounts_utils.AccountsUtils(
        logger=self.logger, groups=groups, remove=remove,
        useradd_cmd=useradd_cmd or DEFAULT_USERADD_CMD,
        userdel_cmd=userdel_cmd or DEFAULT_USERDEL_CMD,
        usermod_cmd=usermod_cmd or DEFAULT_USERMOD_CMD,
        groupadd_cmd=groupadd_cmd or DEFAULT_GROUPADD_CMD)
```